src/container/__tests__/RelayContainer_setVariables-test.js

```diff
@@ -292,6 +292,15 @@ describe('RelayContainer.setVariables', function() {
       expect(mockInstance.state.queryData.entity).toBe(updatedQueryData);
     });
 
+    it('re-renders when there are pendingVariables', () => {
+      const before = render.mock.calls.length;
+      expect(render.mock.calls.length).toBe(1);
+      mockInstance.setVariables({site: 'www'});
+      jest.runAllTimers();
+      environment.primeCache.mock.requests[0].block();
+      expect(render.mock.calls.length - before).toBe(1);
```

Comment:
I added some logging statements as well as a second `runAllTimers`:

```javascript
    it('re-renders when there are pendingVariables', () => {
      render.mockClear();
      console.log(render.mock)
      mockInstance.setVariables({site: 'www'});
      console.log(render.mock)
      jest.runAllTimers();
      console.log(render.mock)
      environment.primeCache.mock.requests[0].block();
      console.log(render.mock)
      console.log(render.mock.instances[0].props)
      expect(render.mock.calls.length).toBe(1);
    });
```

Here is the output 
```
{ calls: [], instances: [] }
{ calls: [], instances: [] }
{ calls: [], instances: [] }
{ calls: [ [] ],
  instances:
   [ { props: [Object],
       context: {},
       refs: {},
       updater: [Object],
       state: null,
       _reactInternalInstance: [Object] } ] }
{ entity:
   { __dataID__: '42',
     __fragments__: { '0::client': '42' },
     id: '42',
     url: '//url',
     profilePicture: { uri: '//url' } },
  relay:
   { applyUpdate: [Function: bound applyUpdate],
     commitUpdate: [Function: bound commitUpdate],
     forceFetch: [Function: bound forceFetch],
     getPendingTransactions: [Function: bound getPendingTransactions],
     hasFragmentData: [Function: bound hasFragmentData],
     hasOptimisticUpdate: [Function: bound hasOptimisticUpdate],
     hasPartialData: [Function: bound hasPartialData],
     pendingVariables: { site: 'www' },
     route: MockRoute { name: 'MockRoute', params: {}, queries: {}, uri: [Getter] },
     setVariables: [Function: bound setVariables],
     variables: { site: 'mobile' } } }
```

The output ends up being the same with or without the udpates in RelayContainer.js, I'm puzzled